src/sentry/celery.py

```diff
@@ -44,7 +41,9 @@ def holds_bad_pickle_object(value, memo=None):
             "django database models are large and likely to be stale when your task is run. "
             "Instead pass primary key values to the task and load records from the database within your task.",
         )
-    if type(value).__module__.startswith(("sentry.", "getsentry.")):
+    app_module = type(value).__module__
+    ok_class = isinstance(value, Enum)
```

Comment:
Is it ok to pickle enums? These can still break if they are moved around in the codebase

Reply:
Yeah, they could move around. I added this because a `buffer_incr` call for Group includes `group.data` which contains a nested enum value - `data.metadata.initial_priority`. That value will encode to JSON cleanly, and is stored in postgres as an integer as well. I felt like this was the less invasive change instead of reworking the `buffer_incr` call within `EventManager`.

Reply:
I updated the callsite instead in https://github.com/getsentry/sentry/pull/87863/commits/f66bf56cbc9a67b798fd34bfb7f9e6a60d94416c